### DISCUSSION TITLE
Add Queries

### DISCUSSION THREAD

**Discussion by fuzolan** [open-metadata/OpenMetadata#10630] [Q&A] [answered]
The Queries-Tab says "Go ahead and add a new queries!" But there is no direct information about it in the documentation neither in the application.  If the connector like oracle is not compatible with it, then maybe it's better to hide the tab or give better information about it. Nevertheless it might be nice to manually add queries to the specific schema

**Comment by pmbrull** [comment, marked as answer]
Hi @fuzolan, queries are automatically ingested during the Usage Workflow ([docs](https://docs.open-metadata.org/connectors/ingestion/workflows/usage)).

On 1.0 we will also add the possibility of adding queries manually in case there is something you'd like to share with your peers.

Hope this helps!
